aptible-auth.gemspec

```diff
@@ -21,9 +21,10 @@ Gem::Specification.new do |spec|
   spec.require_paths = ['lib']
 
   spec.add_dependency 'aptible-resource', '~> 1.0'
+  spec.add_dependency 'concurrent-ruby', '1.3.4'
   spec.add_dependency 'gem_config'
   spec.add_dependency 'multipart-post', '2.1.1'
-  spec.add_dependency 'oauth2', '1.4.7'
+  spec.add_dependency 'oauth2', '2.0.9'
```

Comment:
I wanted to add 3.x since that's what our APIs are on, but that breaks rubocop, and I don't really want to deal with that when all I'm trying to do is make what should be a minor version bump